h2o-docs/src/product/performance-and-prediction.rst

```diff
@@ -1620,15 +1620,15 @@ Examples:
 
         # set the predictors and response columns:
         predictors <- c("economy", "cylinders", "displacement", "power", "weight")
-        response = "acceleration"
+        response = "acceleration" # Stopping metric cannot be mean_per_class_error for this continous target variable
```

Comment:
Good catch! @hannah-tillman can turn the example to a classification problem? (E.g.: https://docs.h2o.ai/h2o/latest-stable/h2o-docs/data-science/algo-params/training_frame.html)